mlir/test/Dialect/Rock/integration/multibuffer/lit.local.cfg

```diff
@@ -0,0 +1,2 @@
+if "gfx1030" in config.arch:
```

Comment:
I don't think this is the right way to phrase the exclude. Also, why is it here?

Reply:
This is because I have an `accel_gemm` operator which would not work for non accelerated GPUs

Reply:
Ah

Reply:
... you could test for wmma or mfma in the default feature set instead?

Reply:
Done (I think)